packages/openapi-v3/package.json

```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@loopback/context": "^1.24.0",
     "@loopback/repository-json-schema": "^1.11.2",
+    "@loopback/core": "^1.11.0",
```

Comment:
If you need things from `core` that are not part of `context`, then please remove the dependency on `@loopback/context` on L9 and rework imports to use `@loopback/core` instead.